{"name": "Логин с неверными учетными данными", "status": "passed", "parameters": [{"name": "test_cases_input", "value": "{'email': 'wrongexample.com', 'password': 'password'}"}], "start": 1741824935903, "stop": 1741824936002, "uuid": "14d319c9-cd23-476d-95bb-190a4257a600", "historyId": "e2cac74bafd3fec2307b2f8078157b1f", "testCaseId": "cf44ea55174a74afa9efcb95a068cfd0", "fullName": "tests.test_login_user.TestLogin#test_login_invalid_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "TerekhovHONOR"}, {"name": "thread", "value": "18088-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}